src/Samples/DIPS.Xamarin.UI.Samples/Controls/Popup/ViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Extensions;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Popup
+{
+    public class ViewModel : INotifyPropertyChanged
```

Comment:
Follow the other sample view model names. Think this one should be called `PopupPageViewModel`